rotkehlchen/tests/utils/dataimport.py

```diff
@@ -697,133 +698,110 @@ def assert_nexo_results(rotki: Rotkehlchen):
     warnings = rotki.msg_aggregator.consume_warnings()
     errors = rotki.msg_aggregator.consume_errors()
     assert len(errors) == 0
-    assert len(warnings) == 0
+    assert len(warnings) == 6
 
```

Comment:
And one more question: does the new csv file check all the cases? (I mean cases when something is off)

Reply:
Nexo simply changed the column headers to `Amount, Currency` -> `Input Amount, Input Currency, Output Amount, Output Currency`.